sections/advanced/theming.md

```diff
@@ -109,6 +111,22 @@ class MyComponent extends React.Component {
 export default withTheme(MyComponent);
 ```
 
+#### via `useContext` React hook | v4
+
+You can also use `useContext` to access the current theme outside of styled components when working with React Hooks.
+
+```jsx
+import { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+
+const MyComponent = () => {
+  const themeContext = useContext(ThemeContext);
+
+  console.log('Current theme: ', this.props.theme);
```

Comment:
Woops nevermind that approval, this still needs to be fixed. It's `themeContext` not `this.props.theme`.